contracts/Challenge/IChallengeFactory.sol

```diff
@@ -0,0 +1,5 @@
+pragma solidity ^0.4.24;
+
+interface IChallengeFactory {
+  function create(address registry, address challenger, address applicant) returns (address challenge);
```

Comment:
why did you change the argument to `applicant`? `itemOwner` makes a lot more sense to me

Reply:
@emaG3m that makes sense, I changed it back